frontend/src/stories/fixtures/ActivityDashBoardBargraphData.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "id": "date=2022-08-12&event_type=1&filter_by_actor_user=2&filter_by_effected_user=",
+    "date": "2022-08-12",
+    "total": 6,
+    "filter_by_actor_user": "2",
+    "filter_by_effected_user": "",
+    "event_type": 1,
+    "event_type_name": "COMPETENCE_REVIEW_DONE"
```

Comment:
This data is coming from the correct api. Please look at it closely. The event types are well defined, you dont need to make up new ones. They all exist and are all associated with colors already. 

Please remember that the naming convention should be likeThis and not_like_this.  The apis are developed in Python, but the react app converts the names so that they are consistent with js. Please use js names.